From 17992e6924d3cef9d0995b7835d702b4ca6b52e1 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Mon, 24 Jun 2019 14:26:52 +0800
Subject: [PATCH 2/2] do not load the system-wide Xresources

- In https://github.com/rhinstaller/anaconda, revert the
  following commit:
-----------------------------------------------
commit ef85e16c80609efc6bdd50c3a5aa6333fcdf0a57
Author: David Shea <dshea@redhat.com>
Date:   Wed Mar 9 14:43:03 2016 -0500

    Load the system-wide Xresources (#1241724)
-----------------------------------------------

The reason is OE does not have spice-vdagent.

- Use terminal vt7 to display X

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase for anaconda 42 on 20250328. Function do_extra_x11_actions() has
been removed, so drop the related modification at same time.

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
 pyanaconda/display.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/pyanaconda/display.py b/pyanaconda/display.py
index e0d262c075..7ef61a1a95 100644
--- a/pyanaconda/display.py
+++ b/pyanaconda/display.py
@@ -222,8 +222,8 @@ def do_startup_wl_actions(timeout, headless=False, headless_resolution=None):
         # headless (remote connection) - stay on VT1 where connection info is
         argv.extend(["--vt", "1"])
     else:
-        # local display - switch to VT6 & show GUI there
-        argv.extend(["--vt", "6"])
+        # local display - switch to VT7 & show GUI there
+        argv.extend(["--vt", "7"])
 
     # add the generic GNOME Kiosk invocation
     argv.extend(["gnome-kiosk", "--sm-disable",
-- 
2.7.4

